src/poliastro/twobody/orbit.py

```diff
@@ -469,31 +469,46 @@ def from_sbdb(cls, name, **kargs):
 
         obj = SBDB.query(name, full_precision=True, **kargs)
 
-        a = obj["orbit"]["elements"]["a"].to(u.AU) * u.AU
-        ecc = float(obj["orbit"]["elements"]["e"]) * u.one
-        inc = obj["orbit"]["elements"]["i"].to(u.deg) * u.deg
-        raan = obj["orbit"]["elements"]["om"].to(u.deg) * u.deg
-        argp = obj["orbit"]["elements"]["w"].to(u.deg) * u.deg
-
-        # Since JPL provides Mean Anomaly (M) we need to make
-        # the conversion to the true anomaly (\nu)
-        nu = M_to_nu(obj["orbit"]["elements"]["ma"].to(u.deg) * u.deg, ecc)
+        try:
+            obj["count"]
+            # no error till now ---> more than one object has been found
+            objects_name = obj["list"]["name"]  # contains all the name of the objects
+            objects_name_in_str = (
+                ""
+            )  # used to store them in string form each in new line
+            for i in objects_name:
+                objects_name_in_str += i + "\n"
 
-        epoch = time.Time(obj["orbit"]["epoch"].to(u.d), format="jd")
+            raise ValueError(
+                str(obj["count"]) + " different objects found: \n" + objects_name_in_str
+            )
 
-        ss = cls.from_classical(
-            Sun,
-            a,
-            ecc,
-            inc,
-            raan,
-            argp,
-            nu,
-            epoch=epoch.tdb,
-            plane=Planes.EARTH_ECLIPTIC,
-        )
+        except KeyError:
```

Comment:
I don't like this way of using an error to control the flow. Perhaps could do instead:

```
if "count" in obj:
    # Handle the error
    raise ValueError(...)

# No need of else block, just do the rest of the stuff
a = obj["orbit"][...]
...
```

Reply:
I will take care of these changes asap! The idea is pretty much the same as in issue #609, right? Raise a proper message (not a key) when an error is produced.

Reply:
Correct!